models/fixtures/repo_unit.yml

```diff
@@ -550,3 +550,24 @@
   repo_id: 53
   type: 1
   created_unix: 946684810
+
+-
+  id: 81
+  repo_id: 54
+  type: 4
+  config: "{}"
+  created_unix: 946684810
+
+-
+  id: 82
+  repo_id: 54
+  type: 5
+  config: "{}"
+  created_unix: 946684810
+
+-
+  id: 83
+  repo_id: 54
+  type: 1
+  config: "{}"
```

Comment:
And this is apparently optional

```suggestion
```

Reply:
I don't think it's optional. An empty string cannot be unmarshaled.

Reply:
Odd, it's just that

https://github.com/go-gitea/gitea/commit/36a2d2f919ee445f0374c210309b336e7c3c1f8e#diff-1d9ee1e82b5f1037429ca68ae776a814c8361c1258fc505141932b802c3a3a0cR549-R552

disagrees. Once I get the core tests actually working from a clean build I'll test if I can shave this off or not.

Reply:
(CI passes without this line)